### ISSUE TITLE
All players disconnected when a milestone is reached

### ISSUE THREAD

**Issue by Romejanic** [Romejanic/time-wasted-meter#2]
Example error
```
Otheris_ lost connection: Internal Exception: java.lang.NullPointerException: Cannot invoke "String.length()" because "s" is null
```

**Comment by Romejanic** [comment]
Issue is caused by a bug in either Paper or ApexHosting, a temporary fix can be made by implementing a custom broadcast function.

**Comment by Romejanic** [comment]
Fixed in version 1.1.2
